catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/query/handlers/CqlTransformHandlerTest.java

```diff
@@ -0,0 +1,207 @@
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.ServiceReference;
+import spark.Request;
+import spark.Response;
+
+public class CqlTransformHandlerTest {
+
+  private List<ServiceReference> queryResponseTransformers;
+  private CqlTransformHandler cqlTransformHandler;
+  private BinaryContent binaryContent;
+
+  @Mock private ServiceReference<QueryResponseTransformer> mockQueryResponseTransformer;
+  @Mock private BundleContext mockBundleContext;
+  @Mock private EndpointUtil mockEndpointUtil;
+  @Mock private Request mockRequest;
+  // @Mock private Response mockResponse;
+  @Mock private CqlQueryResponse mockCqlQueryResponse;
+  @Mock private QueryResponse mockQueryResponse;
+  @Mock private QueryResponseTransformer mockServiceReference;
+  @Mock private ServletOutputStream mockServletOutputStream;
+  @Mock private HttpServletResponse mockHttpServiceResponse;
+
+  private static final String GZIP = "gzip";
+  private static final String NO_GZIP = "";
+  private static final String QUERY_PARAM = ":transformerId";
+  private static final String RETURN_ID = "kml";
+  private static final String OTHER_RETURN_ID = "xml";
+  private static final String MIME_TYPE = "application/vnd.google-earth.kml+xml";
+  private static final String SAFE_BODY =
+      "{\"src\":\"ddf.distribution\",\"start\":1,\"count\":250,\"cql\":\"anyText ILIKE '*'\",\"sorts\":[{\"attribute\":\"modified\",\"direction\":\"descending\"}],\"id\":\"7a491439-948e-431b-815e-a04f32fecec9\"}";
+  private static final String CONTENT = "test";
+  private static final String SERVICE_NOT_FOUND = "{\"message\":\"Service not found\"}";
+  private static final String SERVICE_SUCCESS = "";
+  private static final String ATTACHMENT_REGEX =
+      "^attachment;filename=export-\\d\\d\\d\\d-\\d\\d-\\d\\dT\\d\\d:\\d\\d:\\d\\d\\.\\d\\d\\dZ."
+          + RETURN_ID
+          + "$";
+
+  private class MockResponse extends Response {
+
+    private String contentType = "";
+    private int statusCode = 0;
+    private Map<String, String> headers = new HashMap<>();
+    private HttpServletResponse mockHttpServletResponse;
+
+    MockResponse(HttpServletResponse mockHttpServletResponse) {
+      this.mockHttpServletResponse = mockHttpServletResponse;
+    }
+
+    @Override
+    public void type(String contentType) {
+      this.contentType = contentType;
+    }
+
+    @Override
+    public String type() {
+      return this.contentType;
+    }
+
+    @Override
+    public void status(int statusCode) {
+      this.statusCode = statusCode;
+    }
+
+    @Override
+    public int status() {
+      return this.statusCode;
+    }
+
+    @Override
+    public void header(String header, String value) {
+      headers.put(header, value);
+    }
+
+    public Map<String, String> getHeaders() {
+      return this.headers;
+    }
+
+    @Override
+    public HttpServletResponse raw() {
+      return this.mockHttpServletResponse;
+    }
+  }
+
+  private MockResponse mockResponse;
+
+  @Before
```

Comment:
❓ This might want to go into @BeforeClass since @Before will run it before every unit test and unnecessarily increase build times.

Reply:
beforeclass can only run if the method is static